test/e2e/install/install_test.go

```diff
@@ -71,14 +71,17 @@ var _ = Describe("[Install] continuousIntegration", func() {
 
 	Context("with a running cluster with all the components", func() {
 		It("[test_id:47574][tier0] should perform overall deployment and verify the condition is reported as available", func() {
-			deployedObj := deploy.OverallDeployment()
-			nname := client.ObjectKeyFromObject(deployedObj.NroObj)
+			deployer := deploy.NewForPlatform(configuration.Plat)
```

Comment:
why don't we expose the object anymore again?

Reply:
The deployer is an interface so we'll have to expose a new `get` and `set` functions for NRO and I didn't see how it fit within the existing interface architectural wise

Reply:
right, but (in tehory, let's discuss before rushing to the change) `Deploy` can be changed to return the object, can't it?

Reply:
It's possible indeed. I had that in mind as well but for some reason I don't recall at the moment I thought it's a bad idea. let me play with that a little further

Reply:
OK I change it to return the object lets see if CI is happy